cgmanifests/submodules/cgmanifest.json

```diff
@@ -100,6 +100,28 @@
         "comments": "manylinux dependency"
       }
     },
+    {
```

Comment:
perhaps these should be added to cgmanifests/cgmanifest.json instead as they are not git submodules

Reply:
> perhaps these should be added to cgmanifests/cgmanifest.json instead as they are not git submodules

That file contains *only* submodules. The file I have updated contains `other` kinds of artifacts.

Reply:
Indeed,  here "other" means the other things that we couldn't handle well.

Reply:
cgmanifests/cgmanifest.json contains non-submodule entries as well. I suppose it's a bit mixed up now, but originally the intent was for cgmanifests/submodules/cgmanifest.json to contain git submodule entries, hence the "submodules" in the path.
https://github.com/microsoft/onnxruntime/blob/master/cgmanifests/README.md

Reply:
How do I auto-regenerate it? The script apparently handles only trivial GIT type entries.

Reply:
```
$ python cgmanifests/submodules/generate_submodule_cgmanifest.py > cgmanifests/submodules/cgmanifest.json
```
that should generate it.
it handles the project's git submodules and parses some dependencies from one of the dockerfiles